### PR TITLE
Clean Composer dependencies

### LINKED ISSUE #772: :arrow_down: victoire requires a way too much dependencies and must be cleaned
## Description
it must not require:
- [ ] infinite-networks/form-bundle": "dev-master
- [ ] jms/translation-bundle
- [ ] jms/di-extra-bundle
- [ ] jms/serializer-bundle
- [ ] troopers/assetic-injector-bundle

## Resolution Ideas
- just remove it if it's just some debt
- move the usefull parts of code directly inside victoire sources

### PR DESCRIPTION
**Pull request by alexislefebvre**
## Type
Feature

## Purpose
- Remove unnecessary dependencies
- Remove `composer.lock`

Fixes #772.

## BC Break
NO

### CODE REVIEW

File: composer.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+| predis/predis                        | ^1.1         |                             |
+| sensio/distribution-bundle           | ^2.3         |                             |
+| sensio/framework-extra-bundle        | ~3.0         |                             |
+| snc/redis-bundle                     | ~2.0         |                             |
+| stof/doctrine-extensions-bundle      | ~1.2         |                             |
+| symfony/assetic-bundle               | ~2.3         |                             |
+| symfony/monolog-bundle               | ~2.4 or ~3.0 |                             |
+| symfony/symfony                      | ^2.8         |                             |
+| symfony/swiftmailer-bundle           | ^2.3         |                             |
+| troopers/alertify-bundle             | ^3.0         |                             |
```
Comment (paulandrieux) [inline composer.md]: Is this bundle really necessary ?

File: Tests/Features/Context/FeatureContext.php [hunk 2]
```diff
@@ -92,4 +93,16 @@ public function theTitleShouldBe($title)
...
     }
+
+    /**
+     * Returns fixed step argument (with \\" replaced back to ").
+     *
+     * @param string $argument
+     *
+     * @return string
+     */
+    protected function fixStepArgument($argument)
```
Comment (lenybernard) [inline Tests/Features/Context/FeatureContext.php]: me neither, it's a mistake

File: composer.md (hunk 1 above)
Comment (lenybernard) [inline composer.md]: We could avoid to use Alertify-bundle but for now it's in every controller:

- https://github.com/Victoire/victoire/search?utf8=%E2%9C%93&q=alertify
-https://github.com/Victoire/victoire/blob/648350a6fe3ced8e615528ef4af14b91b0c1ce54/Bundle/CoreBundle/Controller/VictoireAlertifyControllerTrait.php
- https://github.com/Victoire/victoire/search?utf8=%E2%9C%93&q=%24this-%3Econgrat%28&type=Code

Should we ?

Comment (paulandrieux) [inline composer.md]: Yep, no need for an external lib to do this. The assetic-injector-bundle is unnecessary too. We can do it in a separate PR however.
